testsuite/MDAnalysisTests/coordinates/test_pdb.py

```diff
@@ -243,6 +243,30 @@ def test_check_coordinate_limits_max(self):
         assert_raises(ValueError, u.atoms.write, self.outfile)
         del u
 
+    @attr('issue')
+    def test_check_header_title_multiframe(self):
+        """Check whether HEADER and TITLE are written just once in a multi-
+        frame PDB file (Issue 741)"""
+        u = mda.Universe(PSF,DCD, permissive=True) 
+        pdb = mda.Writer(self.outfile, multiframe=True)
+        protein = u.select_atoms("protein and name CA")
+        for ts in u.trajectory[:5]:
+            pdb.write(u)
+        pdb.close()
+
+        with open(self.outfile) as f:
+            got_header = 0
+            got_title = 0
+            for line in f:
+                header = line[:6]
```

Comment:
You are right. I didn't look upon that function. I will do the changes and then commit.